app/services/activities/import_from_csv.rb

```diff
@@ -113,6 +120,7 @@ def sub_level_b_column_headings
           "DFID policy marker - Gender",
           "DFID policy marker - Nutrition",
           "ODA Eligibility Lead",
+          "Partner organisation identifier",
```

Comment:
I'd like to check with the BEIS team whether this should be included in the template CSV or not - I think it should be included but not required, otherwise people uploading the CSV won't know it's something they can do. I'd like to confirm this with BEIS though.